lib/services/local/access_list_test.go

```diff
@@ -187,6 +191,10 @@ func TestAccessListMembersCRUD(t *testing.T) {
 	require.NoError(t, err)
 	require.Empty(t, cmp.Diff(accessList1Member2, member, cmpOpts...))
 
+	// Add access list member for non existent list should produce an error.
+	_, err = service.UpsertAccessListMember(ctx, newAccessListMember(t, "non-existent-list", "nobody"))
+	require.True(t, trace.IsNotFound(err))
```

Comment:
If this fails, the error message will be something like `false is not true`. You won't know if the failure is because we didn't get an error at all or if it's because we got the wrong type of error.

I'd expand on the error message to make troubleshooting easier.